### ISSUE TITLE
Explicit declaration of Vector for iteration does not compile

### ISSUE THREAD

**Issue by drahnr** [djc/askama#59]
```
{% for x in ["a", "b"] %}
{{ x }}
{% endfor %}
```
nor
```
{% let xes = ["a", "b"] %}
{% for x in xes %}
{{ x }}
{% endfor %}
```
works, which would be intuitively what I'd want to do.

This is mostly for shortening a template where huge parts are redundant, but do not need to be adjustable from within the binary.

**Comment by djc** [comment]
Thanks for the report! I've fixed this, so it should work with latest master.

**Comment by drahnr** [comment]
There is one special case which is not working yet:
```
["", "pr-"]
```

**Comment by djc** [comment]
Right, the empty string was not handled correctly. Should be fixed with 14beb21.
